frameworks/Java/blade/src/main/java/hello/model/World.java

```diff
@@ -0,0 +1,18 @@
+package hello.model;
+
+import com.blade.jdbc.annotation.Table;
+import com.blade.jdbc.core.ActiveRecord;
+
+/**
+ * World model
+ *
+ * @author biezhi
+ * @date 2017/9/22
+ */
+@Table("world")
+public class World extends ActiveRecord {
```

Comment:
I took a look at [ActiveRecord](https://github.com/lets-blade/blade-jdbc/blob/master/src/main/java/com/blade/jdbc/core/ActiveRecord.java); it's pretty interesting.